packages/react/src/use-feature-flag.ts

```diff
@@ -1,20 +1,62 @@
-import { Client, EvaluationDetails, FlagValue, ProviderEvents } from '@openfeature/web-sdk';
-import { useEffect, useState } from 'react';
+import { Client, EvaluationDetails, FlagEvaluationOptions, FlagValue, ProviderEvents, ProviderStatus } from '@openfeature/web-sdk';
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
 import { useOpenFeatureClient } from './provider';
 
-export function useFeatureFlag<T extends FlagValue>(flagKey: string, defaultValue: T): EvaluationDetails<T> {
-  const [, setForceUpdateState] = useState({});
+type ReactFlagEvaluationOptions = {
```

Comment:
I've defaulted both of these to true. I think that's what people would want. Reasoning:

- suspending if you don't have `<Suspend>` doesn't really have a negative impact
- if you are getting change events in the UI, you have probably set up your provider to do that, and therefor you probably want to update the UI. If your provider never gets readltime changes, then it's no problem.